src/utils/Xhr.js

```diff
@@ -92,6 +101,12 @@ class Xhr {
 
         this.axios.interceptors.response.use(this.responseInterceptor, this.errorInterceptor);
 
+        if (language) {
```

Comment:
I'm going to make a method that assembles headers from the options object, instead of this